client/src/frontend/components/resource-collection/Description/index.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import FormattedDate from "global/components/FormattedDate";
+import * as Styled from "./styles";
+
+export default function Description({ date, description }) {
+  const { t } = useTranslation();
+
+  if (!date && !description) return null;
+
+  return (
+    <Styled.Wrapper>
+      {date && (
+        <Styled.DateWrapper>
+          <FormattedDate
+            prefix={t("dates.collection_created")}
+            format="MMMM yyyy"
+            date={date}
+          />
```

Comment:
Make sure to use a valid format for the `datetime` attribute (needs dashes not slashes). https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time#valid_datetime_values
<img width="558" alt="Screen Shot 2022-04-21 at 10 43 07 AM" src="https://user-images.githubusercontent.com/12454203/164519747-63e172f5-b560-40af-9daa-49113ff6cb2d.png">

Reply:
This seemed hard. Both `date-fns` and `Date.toLocaleDateString()` use slashes. I couldn't come up with a programmatic way to get localized format and dashes. Ended up writing this function:
```
const dateTime = () => {
    try {
      let dateTimeFormat;
      switch (format) {
        case "MMMM, yyyy":
        case "MMMM yyyy":
          dateTimeFormat = "M yyy";
          break;
        case "PPpp":
          return parsedDate.toISOString();
        default:
          dateTimeFormat = "P";
          break;
      }
      return formatDate(parsedDate, dateTimeFormat, {
        locale: t("date_fns", { returnObjects: true })
      })
        .replaceAll("/", "-")
        .replaceAll(" ", "-");
    } catch {
      return "";
    }
  };
```
Since we only use a few formats in Manifold, it seems to work, but it also feel fragile. Let me know if you have other ideas!

Reply:
I agree, this is probably too brittle. If you look at that [MDN doc I mentioned](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time#valid_datetime_values), it doesn't have to be in a `03-03-2022` format. There are a lot of possibilities, just not slashes. I assume one of the other options will be easier to achieve?